README.md

```diff
@@ -39,15 +39,15 @@ A youtube gameplay video can be found here: https://www.youtube.com/watch?v=-PEK
 * Changes in UI
   * We have added an instructions scene and a stunning menu with automatic gameplay on the background
 * Changes in logic
-  * We have fixed uncountable bugs which made the game unplayable
+  * We have fixed a very countable number of bugs
```

Comment:
What we might get out is the word 'uncountable' from the initial phrase. Makes it a bit milder.